codechain/rpc.rs

```diff
@@ -64,6 +66,37 @@ pub fn setup_http_rpc_server(
     }
 }
 
+#[derive(Debug, PartialEq)]
+pub struct IpcConfiguration {
+    pub socket_addr: String,
+}
+
+impl Default for IpcConfiguration {
```

Comment:
Please remove it unless it's necessary. The hard-coded path for default value seems not great.